test/e2e/introduction.test.js

```diff
@@ -0,0 +1,15 @@
+const { assert, driver } = require('vl-ui-core').Test.Setup;
+const VlIntroductionPage = require('./pages/vl-introduction.page');
+
+describe('vl-introduction', async () => {
+    const vlIntroductionPage = new VlIntroductionPage(driver);
+
+    before(() => {
+        return vlIntroductionPage.load();
+    });
+
+    it('als gebruiker kan in de introductie lezen', async () => {
+        const introduction = await vlIntroductionPage.getIntroduction();
+        await assert.eventually.equal(introduction.getText(), 'Nulla vitae elit libero, a pharetra augue. Sed posuere consectetur est at lobortis. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Sed posuere consectetur est at lobortis. Etiam porta sem malesuada magna mollis euismod. Vivamus sagittis lacus vel augue laoreet rutrum faucibus.');
```

Comment:
moet ge niet eerder de class testen hier dan den text? component doet niks met den text ...

Reply:
Nee, dat is scope van de unit test. E2E gaat geen enkele afnemer hier in geïnteresseerd zijn.

Reply:
ca va, op zich in de text ook niet concreet maar nie erg

Reply:
Ja, het was maar om iets te hebben in de E2E test, `isDisplayed` had hier zeker ook gekund.